### PR TITLE
[MGW-238] Tweak documentation to clarify service catalog limitation

### PR DESCRIPTION
**Pull request by leobellaera**
[Jira Ticket](https://www.mulesoft.org/jira/browse/MGW-238)

### CODE REVIEW

**Review by dosten — DISMISSED** [review]
LGTM, let's wait for Priya's approval

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc
Comment (dukesphere) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: we usually avoid tenses like "have install"

The prerequisite are:
* Install x

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc
Comment (dukesphere) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Prob say separately "Do not install" 

And prob should tell them what to do if they have already installed the Service Catalog. Uninstall? Anything else?

**Review by dukesphere — DISMISSED** [review]
some style and substantive suggestions

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

**Review by dukesphere — APPROVED** [review]

**Comment by leobellaera** [comment]
@dosten Do you think it's ok?

**Comment by dosten** [comment]
@leobellaera @dukesphere it's ok from my side

**Pull request merged**
